pyOCD/tools/flash_tool.py

```diff
@@ -144,12 +154,42 @@ def main():
             if args.hide_progress:
                 progress = None
 
+            # Convert address and count to int.
+            if args.address:
+                args.address = int(args.address, base=0)
+            if args.count:
+                args.count = int(args.count, base=0)
+
+            has_file = args.file is not None
+
             chip_erase = None
             if args.chip_erase:
                 chip_erase = True
             elif args.sector_erase:
                 chip_erase = False
 
+            if not has_file:
+                if chip_erase:
+                    print("Erasing chip...")
+                    flash.init()
+                    flash.eraseAll()
```

Comment:
Yes, this will certainly lock Kinetis devices. That's actually what I wanted, because it is useful for testing. But perhaps for most users it would be a bad thing. Do you think it should perform a mass erase instead?